src/main/pages/che-7/overview/con_che-server.adoc

```diff
@@ -15,4 +15,7 @@ The {prod-short} server, also known as *wsmaster*, is the central service of the
 
 | Container image
 | `eclipse/che-server`
+
+| Environment variables
+| link:https://www.eclipse.org/che/docs/che-7/advanced-configuration-options-for-the-che-server-component/[Advanced configuration options for the Che server component]
```

Comment:
This is not downstream-friendly. Can we use an attribute here?

Reply:
Ok, will do

Reply:
From what I know from Serhii Leshchenko, currently on PTO, the Controller is not part of CRW yet, so this is pure upstream ATM. I will add the attribute for this link in the comments, so it could be easily used in the future when a CRW counterpart will exist.

Reply:
Even though we have it in the downstream docs already:
https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces/2.1/html/administration_guide/codeready-workspaces-architectural-elements_crw#codeready-workspaces-workspace-controller_crw